internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -367,6 +367,17 @@ func createWindowsContainerDocument(ctx context.Context, coi *createOptionsInter
 		return nil, nil, err
 	}
 
+	// add any device extensions
+	extensionPaths, err := getDeviceExtensionPaths(coi.Spec.Annotations)
+	if err != nil {
+		return nil, nil, err
+	}
+	extensions, err := getDeviceExtensions(ctx, extensionPaths)
```

Comment:
nit: maybe pass in annotations here and let `getDeviceExtensions` call to `getDeviceExtensionPaths`?

Reply:
Done :)